### PR TITLE
Adding support for working with stories from the API client, as well as adding aggregation support in the backend

### PR DESCRIPTION
**Pull request by kiddinn**
This PR does quite a few things:

1. Increments client API version number
2. Adds a Story class that can add views and text to a story
3. Adds the ability to delete a story from a sketch in the API
4. Adds some logging into the API client for failed resource fetches
5. Adds the ability to create and list stories in the sketch object for the API client
6. Adds ability to manipulate and work with stories and individual blocks in the API client
7. Adds the ability for analyzers to add aggregation to stories.
8. Fixes an issue where API client tests weren't run and fixes broken tests

### CODE REVIEW

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -143,6 +144,31 @@ def _build_pandas_dataframe(self, search_response, return_fields=None):
...
+        data = {
+            'title': title,
+            'content': ''
+        }
+
+        response = self.api.session.post(resource_url, json=data)
+        response_json = response.json()
+        story_dict = response_json.get('objects', [{}])[0]
+        return story.Story(
+            story_id=story_dict.get('id', -1),
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py]: -1 ?

File: api_client/python/timesketch_api_client/sketch.py [hunk 2]
```diff
@@ -185,14 +211,38 @@ def get_aggregation(self, aggregation_id):
                 return aggregation_obj
         return None
 
+    def get_story(self, story_id=None, story_title=None):
+        """Returns a story object that is stored in the sketch.
+
+        Args:
+            story_id: an integer indicating the ID of the story to
+                be fetched. Defaults to None.
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py]: Isn't this argument required?

File: api_client/python/timesketch_api_client/sketch.py (hunk 1 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: don't remember why, cjhanged to zero

File: api_client/python/timesketch_api_client/sketch.py (hunk 2 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: no, either one... if need to provide either one, if you provide story_title the first instance of a story with that title will be returned... made that a bit more clear in the documentation.

**Comment by kiddinn** [comment]
responded to comments

File: api_client/python/timesketch_api_client/sketch.py (hunk 2 above)
Comment (berggren) [reply api_client/python/timesketch_api_client/sketch.py]: hmm, is this really what we want? Oh, a Story for that sketch right? that would make sense

Comment (berggren) [reply api_client/python/timesketch_api_client/sketch.py]: Ok, this is for the sketch. All good

**Pull request merged**